tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -369,6 +386,7 @@ def resolve_args(arg_str, context=None, resolve_anon=True, filename=None):
     @rtype:  str
     @raise SubstitutionException: if there is an error resolving substitution args
     """
+    
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.